### DISCUSSION TITLE
How do you make it so /pages/ doesn’t show in the URL?

### DISCUSSION THREAD

**Discussion by PoIygon** [FlagClicked/FlagClicked#85] [Help] [answered]
I have a pages folder in my website and I wanted to know how to make /pages not show in the URL

**Comment by FunctionalMetatable** [comment, marked as answer]
this current site (before nuxtification) is currently hosted as a nodejs application. you cannot use nodejs in gh pages tho.

**Reply by PoIygon** [reply]
> this current site (before nuxtification) is currently hosted as a nodejs application. you cannot use nodejs in gh pages tho.

well that didnt help

**Reply by FunctionalMetatable** [reply]
> > this current site (before nuxtification) is currently hosted as a nodejs application. you cannot use nodejs in gh pages tho.
> 
> well that didnt help

You can try adding nuxt?
